changelogs/unreleased/gh-8825-luajit-fixes.md

```diff
@@ -14,3 +14,7 @@ were fixed as part of this activity:
   fractional part raises error too now.
 * Fixed load forwarding optimization applied after table rehashing.
 * Fixed recording of the `BC_TSETM`.
+* Fixed the panic routine when `mprotect` fails to change flags for mcode area.
+* Fixed frame for on-trace OOM handling.
+* Fix handling of instable types in TNEW/TDUP load forwarding.
+* Handle table unsinking in the presence of `IRFL_TAB_NOMM`.
```

Comment:
"Handled", I suppose? Bullet-list should be grammar-consistent.

Reply:
The same with "Fix" above. _**Fixed**_, thanks ;)